doc/source/api/watershed-api-usage.md

```diff
@@ -0,0 +1,14 @@
+This API endpoint provides contextual information about the watershed that drains to the point specified by the parameters. It is intended to clarify and provide context for data about streamflow in watershed.
+
+Every grid cell is defined as flowing into a single other grid cell, so this data is most reliable for larger watersheds representing at least ten grid cells, and completely inappropriate for creeks or culverts smaller than a single grid cell. At small scales, streamflow variations within grid cells, not capturable by a gridded dataset, play too large a role.
+
+## Hypometric curve
+The `hypsometric_curve` object defines a histogram of area by elevation. 
+
+* Elevation bins are of equal width, `w = elevation_bin_width`.
+* Elevation bin `k` is bounded by elevations `ek` and `ek+1`, where `ek = e0 + (k * w)` for `0 <= k < n`
+* where `e0 = elevation_bin_start` and `n = elevation_num_bins`
+* The sum of areas of all cells with elevation falling into elevation bin `k` is given by `ak = cumulative_areas[k]`.
+* Units of elevation and area are specified by the properties `elevation_units` and `area_units`.
+
+Because elevation data is given as the mean of each grid square it is possible to have nonzero values for two elevation bins `ek` and `ek+2` but a zero value for `ek+1` in steep landscapes.
```

Comment:
Wot, no LaTeX? Also, I have a fuller description of the problem for steep landscapes. "Source code" below:

```
# Hypsometric curve

The `/streamflow/watershed` API endpoint response includes a hypsometric curve for the watershed that drains to the specified mouth point. This curve is defined by the data in the `"hypsometric_curve"` property of the response JSON.

## Content

The `"hypsometric_curve"` property has the following format:

```json
"hypsometric_curve": {
    "elevation_bin_start": 0,
    "elevation_bin_width": 100,
    "elevation_num_bins": 46,
    "cumulative_areas": [
        // 46 numerical values
    ],
    "elevation_units": "m",
    "area_units": "m2"
}
```

This object defines a histogram of area by elevation.

Elevation bins are of equal width, $w =$ `elevation_bin_width`. 

Elevation bin $b$ is bounded by elevations $e_b$ and $e_{b+1}$, where 

$$
e_b = e_0 + bw \textnormal{ for } 0 \leq b < n
$$

where $e_0 =$ `elevation_bin_start` and $n =$ `elevation_num_bins`

The sum of areas of all cells with elevation falling in to elevation bin $b$ is given by $a_b =$ `cumulative_areas[b]`.

Units of elevation and area are specified by the properties `"elevation_units"` and `"area_units"`.



## Notes

### Gaps in hypsometric curve, or "empty" bins

For large areas of the earth and reasonably large elevation bins, we expect to see non-zero cumulative areas for each elevation bin between the minimum and maximum elevation over that area. In other words, there should be at least some area at each elevation in the histogram.

However, for small areas with steep topography, it is common to see some of the elevation bins between min and max elevation with zero area. This is not an error in either the computation or the data that feeds it. It is instead a product of the fact that $n$ surface grid cells can represent at most $n$ elevations. 

Consider the most extreme case of $n = 2$ cells that happen to be positioned at the edge of a steep sided valley. One cell is in the valley bottom with an average elevation of 100 m. The other is cell, just adjacent to it, mostly covers the highland above with an average elevation of 500 m. In a histogram with 100 m bin width, we'd see non-zero areas for the 100 m bin and the 500 m bin, but zero areas for the 200 m, 300 m, and 400 m elevation bins, and in the graph these would look like gaps. 

We can see a similar effect for other small values of $n > 2$ in steep terrain too. Once $n$ becomes large enough, then the likelihood of an elevation bin not having *some* cells is quite low and these gaps do not appear.
```

Reply:
I have been able to figure out how to export _to_ LaTeX, but not how to render _from_ LaTeX. Though I am not particularly confident I have exhausted all the plugin options yet. 

Technical problems I want to solve and have reasonably hopes they're solvable:
- add links to markdown files that are correctly converted to point to the html files when the output is rendered
- easier build like the pdp has, and testing the buid
- render LaTeX